### PR TITLE
German Language

### PR DESCRIPTION
**Pull request by Smokingsquirrel**
## About The Pull Request
Overrides PR #4731 with cleaned code and hopefully correct GitHub submission and PR.

### CODE REVIEW

**Comment by ACCount12** [comment]
Why?

I've never seen it being requested, I've never seen it in any of the design docs, and I don't think there ever was any pressing need for a new language.

File: code/modules/mob/language/german.dm [hunk 1]
```diff
@@ -0,0 +1,393 @@
+/datum/language/german
+	name = LANGUAGE_NEO-GERMANIC
```
Comment (Kurgis) [inline code/modules/mob/language/german.dm]: This seems like a define, remove the "-".

Comment (Kurgis) [reply code/modules/mob/language/german.dm]: `LANGUAGE_GERMAN` so it conforms with the other languages.

Comment (Smokingsquirrel) [reply code/modules/mob/language/german.dm]: Thank you for the feedback

**Pull request closed without merging**